spec/models/auth_token_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe AuthToken do
+  it 'encodes and decodes tokens' do
```

Comment:
Except it looks like I had to cheat in one spot, but that's because encryption isn't always the same output; so it didn't transfer over as cleanly as here.